### DISCUSSION TITLE
Integrate with DefiLlama "meta aggregator"?

### DISCUSSION THREAD

**Discussion by gpxl-dev** [airswap/airswap-web#734] [Ideas]
Defillama recently launched a new "meta aggregator" [here](https://swap.defillama.com/?chain=arbitrum).

I'd like to do a temperature check for integrating with them, probably as two separate services, AirSwap RFQ and AirSwap Last Look. 

Personally I think aggregators are the way to go for RFQ & Last Look. This is because due to the prevalence of AMMs users have very high expectations that they should be able to receive a quote for almost any token pair. AirSwap makers may not provide this, so the experience when using a dex that sources liquidity from AirSwap only can be jarring and brought to an abrupt end, which is unfortunately likely to stick in a user's mind and may prevent them from checking next time they wish to make a swap.

Aggregators fix this, because they will only really highlight the positive case. When AirSwap presents the best option, it'll appear at the top of the list. Even when it's not top, it's likely rare that users take any notice of the other options.

Anecdotally, DefiLlama's aggregator has recently become the "go-to" of dex aggregators for many power users because of its completeness and its emphasis on accurate gas pricing, so I think would compliment the (probably) more casual user base of metamask swaps.

I understand that there's currently ongoing work to integrate with paraswap, which would automatically include AirSwap in DefiLlama's aggregator results, however these would be labelled as paraswap rather than AirSwap directly, and I would also assume there is a gas overhead of going through them. 

Providing the llamas are ok with an integration here I don't think there's any downside.

**Comment by Warpedpoetic85** [comment]
If we get the nod to integrate with DefiLlama directly, will the integration with Paraswap still be necessary?

**Reply by gpxl-dev** [reply]
No more "necessary" than it ever was, but paraswap of course will have some direct users that we would be excluding if we didn't make that integration.
